core/src/main/java/io/kestra/core/runners/RunContextLogger.java

```diff
@@ -41,6 +41,9 @@ public class RunContextLogger implements Supplier<org.slf4j.Logger> {
     @Getter
     private File logFile;
     private OutputStream logFileOS;
+    @Setter
+    @Getter
+    private Instant customInstant;
```

Comment:
This is not a good idea as the logger is reused for multiple event.
The timestamp must be tied to a specific log event, not to the logger.
If the logger is used concurrently, one timestamp would override another.